lib/ret_web/views/api/v1/avatar_view.ex

```diff
@@ -23,7 +23,8 @@ defmodule RetWeb.Api.V1.AvatarView do
       # Only include account id on your own avatars
       account_id: account && avatar.account_id == account.account_id && avatar.account_id |> Integer.to_string(),
       allow_remixing: avatar.allow_remixing,
-      allow_promotion: avatar.allow_promotion
+      allow_promotion: avatar.allow_promotion,
+      has_listings: length(avatar.avatar_listings) > 0
```

Comment:
i think this needs to filter out delisted avatars, there's probably a fancy ecto way to create a `visible_avatar_listings` association or something we can preload

Reply:
Just did a simple inline filter